packages/carousel/src/Carousel.js

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+import Image from "@crave/farmblocks-image";
+
+import Container from "./styledComponents/Carousel";
+
+const defaultConfig = {
+  width: 656,
+  height: 328,
+  margin: 20,
+  displayTime: 4000,
+  border: {
+    radius: "16px",
+    width: "4px",
+    color: "rgba(255, 255, 255, 0.56)"
+  }
+};
+
+class Carousel extends React.Component {
+  state = {
+    activeItem: 0,
+    transitionInterval: 0
+  };
+
+  nextItem = () => {
+    if (this.state.activeItem + 1 === this.props.imageSet.length) {
+      if (this.state.transitionInterval) {
+        window.clearInterval(this.state.transitionInterval);
+        this.props.onEnd();
+      }
+      return null;
+    }
+    const activeItem = this.state.activeItem + 1;
+    this.props.onChange(activeItem);
+    return this.setState({ activeItem });
+  };
+
+  componentDidMount = () => {
+    const transitionInterval = window.setInterval(
+      this.nextItem,
+      this.props.itemConfig.displayTime
+    );
+    return this.setState({ transitionInterval });
+  };
+
+  render() {
+    const { imageSet, itemConfig } = this.props;
+    return (
+      <Container activeItem={this.state.activeItem} itemConfig={itemConfig}>
```

Comment:
We should merge the default config with `props.itemConfig`.

I did this and it loose the other configs:
```
<Carousel
          onChange={action("photo changed")}
          onEnd={action("end")}
          imageSet={imageSet}
          itemConfig={{
            displayTime: 3000
          }}
        />
```

Rendered this 
![image](https://user-images.githubusercontent.com/17936244/37795868-6f390172-2df4-11e8-943b-2273d1005ffb.png)